### PR TITLE
17.2 stories improvements

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

17.2 stories improvements

-----

### CODE REVIEW

File: stories/documentation/actions/button/html&css/button-only-icon.stories.ts [hunk 1]
```diff
@@ -7,7 +7,10 @@ export default {
 } as Meta;
 
 function getTemplate(args: ButtonIconStory): string {
-	return `<button type="button" class="button mod-onlyIcon"><span aria-hidden="true" class="lucca-icon icon-heart"></span></button>`;
+	return `<button type="button" class="button mod-outlined mod-onlyIcon"><span aria-hidden="true" class="lucca-icon icon-heart"></span></button>
```
Comment (vvalentin-lucca) [inline stories/documentation/actions/button/html&css/button-only-icon.stories.ts:10]: I think there's a need for masked text examples.

**Review by vvalentin-lucca — APPROVED** [review]

**Review by vvalentin-lucca — APPROVED** [review]

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/12123/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2524/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/12125/ui-diff)

**Pull request merged**